okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/UrlSelectorImpl.java

```diff
@@ -38,7 +38,7 @@ private UrlSelectorImpl(ImmutableList<HttpUrl> baseUrls) {
     static UrlSelectorImpl create(Collection<String> baseUrls) {
```

Comment:
can add docs about the baseUrls, now that this is somewhat non-obvious?

Reply:
done